galgebra/ga.py

```diff
@@ -1870,18 +1870,16 @@ def _build_reciprocal_basis(self, gsym):
         else:
             if gsym is not None:
                 # Define name of metric tensor determinant as sympy symbol
-                if printer.GaLatexPrinter.latex_flg:
-                    det_str = r'\det\left ( ' + gsym + r'\right ) '
+                if self.coords is None:
+                    gsym_obj = MatrixSymbol(gsym, self.n, self.n)
                 else:
-                    det_str = 'det(' + gsym + ')'
+                    gsym_obj = MatrixFunction(gsym, self.n, self.n)(*self.coords)
                 # Define square of pseudo-scalar in terms of metric tensor
                 # determinant
+                det_g = Determinant(gsym_obj)
+                det_g._assumptions['commutative'] = True  # sympy/sympy#19308
```

Comment:
This turns out not to be safe :(

Reply:
Why did you need to change this in the first place?

Reply:
Because galgebra assumes any non-commutative term is a basis blade.

Reply:
I see

Reply:
On second thoughts, we should go ahead with making this change anyway, but in a way that doesn't interfere with non-galgebra determinants.

I've updated this PR to do that, and filed an upstream PR at https://github.com/sympy/sympy/pull/19354 to hopefully eliminate the need for the workaround eventually.